clio-client/src/main/scala/org/broadinstitute/clio/client/commands/AddWgsUbamCommand.scala

```diff
@@ -15,10 +15,11 @@ import io.circe.parser._
 import org.broadinstitute.clio.client.webclient.ClientAutoDerivation._
 
 object AddWgsUbamCommand extends Command {
-  def execute(webClient: ClioWebClient, config: BaseArgs)(
-    implicit ec: ExecutionContext
-  ): Future[HttpResponse] = {
-    val decodedOrError = parse(IoUtil.readMetadata(config.metadataLocation.get))
+  def execute(
+    webClient: ClioWebClient,
+    config: BaseArgs
+  )(implicit ec: ExecutionContext, ioUtil: IoUtil): Future[HttpResponse] = {
```

Comment:
I still don't think `ioUtil` should be an implicit, here and elsewhere. The few extra characters it saves us from typing isn't worth the magic in this case.

In general I think implicit args should be avoided unless:
1. An API we're using expects the arg to be implicit (like the Akka stuff that expects an implicit `ActorSystem`, and all the methods on `Future` that expect an implicit `ExecutionContext`), or
2. The API is noticeably cleaner with the parameter as implicit (again looking at the methods on `Future`, they all have signatures like `def xyz(f: X => Y)(implicit ec: EC)`. The fact that the EC is implicit lets us write `fut.xyz { x => x.y }` instead of `fut.xyz(x => x.y, ec)` or `fut.xyz(x => x.y)(ec)`)

`ioUtil` doesn't qualify.